prow/README.md

```diff
@@ -85,6 +85,7 @@ Prow is used by the following organizations and projects:
 - [Azure acs-engine](https://github.com/Azure/acs-engine/tree/master/.prowci)
 - [tensorflow/minigo](https://github.com/tensorflow/minigo#automated-tests)
 - [helm/charts](https://github.com/helm/charts)
+- [daisy(google compute image tools)](https://github.com/GoogleCloudPlatform/compute-image-tools/tree/master/test-infra#prow-and-gubenator)
```

Comment:
super-nit: I think Daisy is typically capitalized.

Reply:
:+1: